lib/prepareOptions.js

```diff
@@ -0,0 +1,22 @@
+module.exports = function prepareOptions(options, argv) {
```

Comment:
@AndersDJohnson What did you mean by the following: 

> We should consider adding staged-only linting for this.

Looking to add an issue for this so a contributor can pick it up (unless you would like to)